src/tests/commentsCounter.test.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-undef */
+/* eslint-disable quote-props */
+import itemsCounter from '../function/itemsCounter.js';
+
+global.comments = [{
+  'item_id': 'item1',
+  'username': 'Jane',
+  'comment': 'Hello',
+},
+{
+  'item_id': 'item1',
+  'username': 'Jane',
+  'comment': 'Hello',
+},
+{
+  'item_id': 'item1',
+  'username': 'Jane',
+  'comment': 'Hello',
+},
+{
+  'item_id': 'item1',
+  'username': 'Jane',
+  'comment': 'Hello',
+}];
+
+describe('shows list counter', () => {
+  test('this list api had four items', () => {
+    // Arrange
+    let count = 0;
+    // Act
+    count = itemsCounter(comments);
+
+    // Assert
+    expect(count).toBe(4);
```

Comment:
- [ ] For the tests, please add more tests to cover the functions you wrote to your app. 😄 🙏 
- [ ] The two test cases you wrote are failing due to this error shown in the image down below.
![Git-Bksh__c_Users_mbakr_OneDrive_Desktop_Work_caps_p1_js_capstone 9_10_2021 7_25_55 PM](https://user-images.githubusercontent.com/47625407/132893977-56790deb-0744-44fc-a5fd-67be0e945467.png)
>If it's passing on your side, please ignore the 2nd comment and focus on adding more tests

Reply:
Test is passing on our side : 
![test](https://user-images.githubusercontent.com/80895497/132895695-dc4e3ab4-46dd-41eb-b3dc-00ec32e62905.png)

Reply:
Then I'm sorry about it 😊🙏🏻 Thank you for communicating ❤️

Reply:
And according to the kanban board tasks, the requirement is to write tests for comments and the likes functions. Can you please clarify on that @mahmoudBakr23.

Reply:
Counters We have counters in all the interfaces that show:

The number of items (home) and The number of comments (comments popup). we create a specific function to calculate these numbers on each page. and we covered it with unit tests using Jest.